en/docs/guides/access-delegation/oauth-token-validation.md

```diff
@@ -0,0 +1,48 @@
+# OAuth Token Validation Using SOAP Service
+
+WSO2 Identity Server provides a SOAP service to validate the OAuth2 token it has issued, which can be used by the resource server. This
+page guides you through calling the SOAP service using [SOAP UI](../../https://www.soapui.org/downloads/latest-release.html).
+
+1.  Add the following configuration property to the `deployment.toml` file in the ` <IS_HOME>/repository/conf` folder to enable admin service WSDLs.
+
+    !!! note "Note"
+        This step is just to make the admin services WSDL accessible. Once the service WSDL is taken, set the property to **false**
```

Comment:
This step is to make the admin services WSDL accessible. Once the service WSDL is accessible, set this property to **false**.